SampleProject/VOKCoreDataManagerTests/DateFormatterTests.m

```diff
@@ -0,0 +1,27 @@
+//
+//  DateFormatterTests.m
+//  SampleProject
+//
+//  Created by Carl Hill-Popper on 4/27/16.
+//  Copyright © 2016 Vokal.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <VOKManagedObjectMap.h>
+
+@interface DateFormatterTests : XCTestCase
+
+@end
+
+@implementation DateFormatterTests
+
+- (void)testDateFormattersLocaleIsPOSIX
+{
+    XCTAssertEqualObjects([VOKManagedObjectMap vok_defaultDateFormatter].locale.localeIdentifier,
+                          VOKDefaultDateFormatterLocaleIdentifier);
+    XCTAssertEqualObjects([VOKManagedObjectMap vok_dateFormatterWithoutMicroseconds].locale.localeIdentifier,
+                          VOKDefaultDateFormatterLocaleIdentifier);
+}
```

Comment:
My instinct is that it'd be better to test the output of the date formatter when applied to various date strings with various current locales, but I'm not sure offhand how to set the current locale programmatically for testing.